MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1451,17 +1555,143 @@ the Select` button. When clicked it closes all the previously selected tabs -->
        it continues to wait for execution of the web page script that caused the slow down. -->
     <string name="slow_script_dialog_action_wait">Wait</string>
 
+    <!-- This string is displayed in the title of the quick PopUp blocking dialog, accessed from the PopUp icon in the URL bar.
+     '%1$s' will be replaced at runtime with the app's name. -->
+    <string name="popup_permission_dialog_message">Popup blocking is %1$s for this site.</string>
```

Comment:
Is the comment accurate here? I believe `'%1$s' will be replaced at runtime with the status of the setting.`.

Reply:
Right. I've updated the comment.